CurrentCard/docs/README.md

```diff
@@ -25,13 +25,13 @@ The high level block diagram of the current sensor card is shown below:
 <img src="images/current-sensor-blockdiagram.svg" width="500%" />
 
 ### Current Sensor
-LEM LA 55-P current sensor is selected for this design, as it is the only sensor available from LEM with an open aperture and PC pins that can measure +/-55A. 
+LEM LA 55-P or LA 100-P current sensors are selected and can be used for this design based on the sensing range desired. They are available from LEM with an open aperture and PC pins. LEM LA 55-P and LA 100-P can measure +/-55A and +/-150A, respectively. Note that the LA100 series also has these variants, LA100-P/SP13 and LA100-TP, which have different gains that users must be careful about when ordering.
```

Comment:
@noguchi-takahiro Can you add the Note about the LA 100 series variants in a separate line with the word "Note" in bold as follows (please refine the language to be better than what I am saying):

**Note:**  The LA 100 series has three variants LA 100-P, LA 100-P/SP13, and LA 100-TP. The sensor gains of each of these variants are different, which has implications in the choice of the burden resistor. The rest of this document is specific to the LA 100-P variant.